### DISCUSSION TITLE
Lack of history in default view of (2023.3) more-info dialog for input booleans

### DISCUSSION THREAD

**Discussion by pauln** [home-assistant/frontend#15798] [Dashboard]
Although the new style of the more-info dialogs introduced in 2023.3 is generally nice, it's resulted in extra clicks to see the logbook/history - which is sometimes the main point of opening the more-info dialog.  For instance, I have some input booleans which are turned on/off by automations - such as tracking the state of my washing machine and tumble dryer based on their power usage; for these devices, the big new "switch" UI in the more-info dialog isn't really helpful, as I can already see the state from the tile/glance/etc card on the dashboard, and I virtually never want to manually change the state.  As such, I have the tap action set to show the more-info dialog - which used to show the logbook/history (with a small "switch" above it which could be used to manually toggle the entity, if I recall correctly)... but now it only shows a giant switch, so in order to see how long my tumble dryer has been running for, I now have to first open the more-info dialog and then hit the small "history" button at the top of the dialog.

A couple of ideas come to mind for ways to remove the extra click:

- Add a "history" action which can be used in place of "more-info", which goes straight to the history view
- Add the duration that it's been in the current state (and/or time that it changed to that state) to the more-info dialog - or perhaps even the entire latest logbook entry - since (at least in my experience) that's the most common thing to be looking for

Has anyone else got any other suggestions, or is there something I've missed which would allow me to get the single-click logbook/history back for my input booleans?  I'd prefer not to add a logbook card to my dashboard for this, since it'd only ever have anything to show on laundry day - the previous version of the more-info dialog gave a nice balance where it's not taking up space on the dashboard but it's only a tap/click away when wanted.

**Comment by Xitee1** [comment]
I also want the old more-info dialog back. Now it is pretty much useless and only shows a huge button. The logs are hidden behind an extra menu. I don't get it because I could just toggle the switch from the lovelace card. What's the point of clicking on the entity just to see the button again, but so huge that there is no space for any relevant info.
And it's not only the switch, the old color picker for lights for example was also way better. Now it is also hidden behind an extra menu.
The new vertival layout of everything is just a huge waste of space. With the vertical design there was space for much more informations.
